Keas.Mvc/Controllers/Api/KeysController.cs

```diff
@@ -160,7 +158,7 @@ public async Task<IActionResult> Create([FromBody]CreateKeyViewModel model)
         }
 
         [HttpPost("{id}")]
-        [ProducesResponseType(typeof(Key), StatusCodes.Status200OK)]
+        [ProducesResponseType(typeof(KeyInfo), StatusCodes.Status200OK)]
```

Comment:
I didn't test it, but this looks like it is returning a key, not keyInfo

Reply:
Should be fixed when I push the latest commit